_topic_maps/_topic_map.yml

```diff
@@ -1862,6 +1862,8 @@ Name: Windows Container Support for OpenShift
 Dir: windows_containers
 Distros: openshift-origin,openshift-enterprise
 Topics:
+- Name: Windows Container Support overview
```

Comment:
The product name for WinC is "Windows Container Support for OpenShift." We have been advised to closely follow this naming exactly by PM. I would say for the overview title, it should follow that official name. We can relax that requirement when we refer to it informally.

Reply:
acked